pkgs/by-name/ne/nexus-mods-app/package.nix

```
@@ -0,0 +1,125 @@
+{ _7zz
+, buildDotnetModule
+, copyDesktopItems
+, desktop-file-utils
+, dotnetCorePackages
+, fetchFromGitHub
+, fontconfig
+, lib
+, libICE
+, libSM
+, libX11
+, nexus-mods-app
+, runCommand
+, enableUnfree ? false # Set to true to support RAR format mods
+}:
+let
+  pname = "nexus-mods-app";
+  version = "0.4.1";
+  src = fetchFromGitHub {
+    owner = "Nexus-Mods";
+    repo = "NexusMods.App";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-FzQphMhiC1g+6qmk/R1v4rq2ldy35NcaWm0RR1UlwLA=";
+  };
+  meta = with lib; {
+    description = "Game mod installer, creator and manager";
+    mainProgram = "NexusMods.App";
+    homepage = "https://github.com/Nexus-Mods/NexusMods.App";
+    changelog = "https://github.com/Nexus-Mods/NexusMods.App/releases/tag/${src.rev}";
+    license = with licenses; [ gpl3 ];
+    maintainers = with maintainers; [ l0b0 ];
+    platforms = platforms.linux;
```

```suggestion
    license = with lib.licenses; [ gpl3 ];
    maintainers = with lib.maintainers; [ l0b0 ];
    platforms = lib.platforms.linux;
```

also `gpl3` is a deprecated identifier, please use `gpl3Only` or `gpl3Plus` in accordance with upstream